app/templates/buyers/preview_brief_source.html

```diff
@@ -54,11 +54,8 @@
 <div class="draft-preview">
   <div class="govuk-grid-row">
     <div class="column-two-thirds">
-      <header class="page-heading-smaller">
         <p class="context">{{ brief.organisation }}</p>
```

Comment:
To make this in line with GOV.UK Design System styles this should be a span

```suggestion
        <span class="govuk-caption-l">{{ brief.organisation }}</p>
```